scripts/install/authelia.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+#
+if [[ ! -f /install/.nginx.lock ]]; then
+    echo_warn "You must install nginx first"
+    exit
+fi
+
+mkdir -p "/opt/authelia"
+wget -qO "/opt/authelia.tar.gz" "https://github.com$(/usr/bin/curl -sNL https://github.com/authelia/authelia/releases | grep -Eom1 '/authelia/(.*)tar.gz')"
+tar -xf "/opt/authelia.tar.gz" -C "/opt/authelia/" authelia-linux-amd64
+ln -fsn "/opt/authelia/authelia-linux-amd64" "/opt/authelia/authelia"
+[[ -f "/opt/authelia.tar.gz" ]] && rm -f "/opt/authelia.tar.gz"
+#
+mkdir -p /etc/authelia
+ex_ip="$(ip -br a | sed -n 2p | awk '{ print $3 }' | cut -f1 -d'/')"
+secret="$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
+password_hash="$(/opt/authelia/authelia hash-password password | awk '{ print $3 }')"
+#
+cat > "/etc/authelia/config.yml" << AUTHELIA_CONF
+host: 127.0.0.1
+port: 9091
+
+server:
+  read_buffer_size: 4096
+  write_buffer_size: 4096
+  path: ""
+
+log_level: debug
+log_file_path: /etc/authelia/authelia.log
+jwt_secret: ${secret}
+default_redirection_url: https://${ex_ip}/login
+
+authentication_backend:
+  disable_reset_password: false
```

Comment:
Would this mean that users can reset their passwords in the login form? That could confuse users when their passwords are different in different places